applications/nrf5340_audio/src/bluetooth/le_audio_bis_headset.c

```diff
@@ -27,7 +29,8 @@ static struct bt_audio_broadcast_sink *broadcast_sink;
 static struct bt_audio_stream streams[CONFIG_LC3_DEC_CHAN_MAX];
 static struct bt_audio_stream *streams_p[ARRAY_SIZE(streams)];
 
-static struct bt_audio_lc3_preset lc3_preset = BT_AUDIO_LC3_BROADCAST_PRESET_48_4_1;
+static struct bt_codec codec =
```

Comment:
Shoud probably add a comment here that BT_AUDIO_LOCATION could be changed to RIGHT in initialize()